### DISCUSSION TITLE
v2.3.0

### DISCUSSION THREAD

**Discussion by mikecao** [umami-software/umami#2111] [General]
### New navigation menu

There is now a navigation menu on the website details page so you can quickly load data related to your website.

![image](https://github.com/umami-software/umami/assets/477975/7d9b5c06-7506-4dee-964d-727cbefcf855)

The realtime page has been moved into the website page for quicker access.

![image](https://github.com/umami-software/umami/assets/477975/77dc1671-43ea-4d4b-bf53-d2d0e30ba451)

### Reports

We're excited to introduce the Reports feature! With the reports infrastructure now in place, we'll be able to quickly add new ways for you to view and analyze your data.

![image](https://github.com/umami-software/umami/assets/477975/35ad5098-e4e5-4074-89a2-28c8378d9248)

This release introduces the Funnel report, which allows you to measure the drop-off of users between pages.

![image](https://github.com/umami-software/umami/assets/477975/acd70dbf-3570-4e66-b844-37c646ce3012)

## Event data

Also under the website page is the event data page where you can view details about your saved custom data.

![image](https://github.com/umami-software/umami/assets/477975/9699258b-8ad4-4f13-b25c-a4b54ab7b228)

You can even  drill into a specific field to see more information.

![image](https://github.com/umami-software/umami/assets/477975/67e362d4-45d7-4200-a399-6798c497666b)

## View only role

We've add a new user role called `View only`. These users will not be able to create websites themselves but can join teams and view their websites.

## New icons

We now show icons for browsers, operating systems and devices.

![image](https://github.com/umami-software/umami/assets/477975/b580702a-889c-4ff0-95b0-f0230935f3e0)

## Changes

- New environment variable `ALLOWED_FRAME_URLS`, which is a space-delimited list of URLs allowed to host the application in an iframe
- Many language updates
- Allow domains such as `.ישראל`

A huge thanks to all the contributors on this release! @franciscao633 @bilguun0203 @valtlfelipe @Maxime-J @RohrerF @cadesalaberry @briancao @MarcHagen @theshamuel @AkashRajpurohit @ammar-madni @tairosonloa @gander @andlil @blyamur @bdcorps @thefourcraft @orangecoloured @abhilakshbansal2001 @jclab-joseph 

<hr /><em>This discussion was created from the release <a href='https://github.com/umami-software/umami/releases/tag/v2.3.0'>v2.3.0</a>.</em>

**Comment by opszen** [comment]
这个版本基于mysql的镜像无法启动，看日志是这个目录 02_report_schema_session_data下的sql冲突了  删除这个目录容器才能正常启动  测了半天

**Reply by franciscao633** [reply]
Can you create an issue with the error logs so we can troubleshoot?

**Reply by opszen** [reply]
![image](https://github.com/umami-software/umami/assets/124872512/25265a60-9f40-4175-b1ab-772d2b961b36)

**Reply by appotry** [reply]
https://github.com/umami-software/umami/issues/2118

just create a issue

**Comment by 4ortytwo** [comment]
I've been checking for this every day! 🤓 Huge thanks to the team! ❤️

**Comment by skipperbent** [comment]
**Users:**
I still don't understand how users are managed. As an admin i'm trying to create a view-only user but can't share my website with a new user..  If the user joins a team with a website, it doesn't show on the users dashboard.

**Event Data**
The new "Event Data" tab seems rushed and useless, only values no information of event-name or way of filtering.

The website favicon also broke with the latest update - calls duckduckgo but returns nothing.

**Comment by tairosonloa** [comment]
Hi @skipperbent, I'm not the maintainer of Umami, but an occasional external contributor, and I thought the same thing about the "Event Data" tab as soon as I tested it out.

I submitted an already merged PR to improve the event data. You can see it [here](https://github.com/umami-software/umami/pull/2127). I guess it will be available in the next Umami release.

**Comment by mvuidev** [comment]
Hi,

Concerning the teams/users, 2 things to improve IMO : 

- Possibility to add a user in a team from the admin account.
- Possibility as a team user to define a default website to show on the dashboard.

Currently, to add a user in a team i have to login as the user and join the team. Not a big deal but could be better.
The second point is more annoying. Have to go to settings > teams > website then [view].

This improvements would be cool to be able to use one instance of Umami for multiple projects/clients.
